domains/auth/Auth.API/Integration.Tests/Controllers/LogoutControllerTests.cs

```diff
@@ -29,15 +33,17 @@ public async Task LogoutAsync_ShouldReturnRedirectToAuthority_WhenInvoked()
         var client = factory.CreateAuthenticatedClient(user, identityToken: identityToken, config: builder => builder.ConfigureTestServices(services => services.AddScoped(_ => oidcOptions)));
 
         var result = await client.GetAsync("auth/logout");
+        var resultContent = await result.Content.ReadAsStringAsync();
+        var redirectUriResponse = JsonConvert.DeserializeObject<RedirectUriResponse>(resultContent);
         Assert.NotNull(result);
```

Comment:
This `assert` should happen before we use the variable. Also the very next thing should be the HTTP status code OK assert.